common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -542,6 +542,10 @@
   "screens.wynntils.wynntilsQuestBook.mainMap.description": "Click here to view the main map.",
   "screens.wynntils.wynntilsQuestBook.mainMap.name": "Main Map",
   "screens.wynntils.wynntilsQuestBook.name": "Wynntils Quest Book",
+  "screens.wynntils.wynntilsQuestBook.questInfo.click": "Click to see Mini-Quests",
+  "screens.wynntils.wynntilsQuestBook.questInfo.click2": "Click to see Quests",
```

Comment:
The `2` at the end is not ideal. Can you rename it like `...questInfo.miniQuests.click` etc?